config.yml

```diff
@@ -26,6 +26,13 @@ layoutDir:              "site/layouts"
 staticDir:              "site/static"
 themesDir:              "site/themes"
 
+module:
+  hugoVersion:
+    min: 0.56.0
+  mounts:
+    - source:           "dist"
+      target:           "site/static/docs/4.3/dist"
```

Comment:
Target must start with one of `static`, `content` etc. They are "virtual folders", so to speak.

Reply:
Also (I'm pretty sure, but I got a little bit overloaded when implementing this): When you create a mounts section, you need to add all you need, e.g. "static => static", "content => content". I.e. is you're specific we don't add the defaults.

Reply:
I tried it with this too but the dist folder isn't copied to the output dir.

```yml
module:
  hugoVersion:
    min: 0.56.0
  mounts:
    - source:           dist
      target:           static/docs/4.3/dist
```

Maybe I'm missing something? The first issue is that we manually need to copy the dist folder into the static folder and generating the SRI hashes is the second one, which like you said, I need to place the files into the assets folder.

Reply:
static is copied, assets is not (only the assets you process and link to). You may create 2 mounts with the same source if you want a combination.

Reply:
Weird, whatever I have tried, the dist folder is never copied for me into the publishDir hence why CI is failing due to the missing assets.

Reply:
I suggest you let this PR cool for a week or so and then ping me. I'm a little bot overloaded in the notification department at the moment. Which is a bad thing considering I'm on vacation :-)